definitions/private-law/json/CaseEvent/CaseEvent.json

```diff
@@ -676,7 +676,7 @@
     "Name": "Manage documents",
     "Description": "Manage documents",
     "DisplayOrder": 38,
-    "PreConditionState(s)": "AWAITING_SUBMISSION_TO_HMCTS;SUBMITTED_PAID;CASE_ISSUED;JUDICIAL_REVIEW;AWAITING_RESUBMISSION_TO_HMCTS;CASE_WITHDRAWN;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
+    "PreConditionState(s)": "SUBMITTED_PAID;CASE_ISSUED;JUDICIAL_REVIEW;AWAITING_RESUBMISSION_TO_HMCTS;CASE_WITHDRAWN;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
```

Comment:
To Reviewer,
This manage Document is not in use, though I have changed pre condition state,I feel this part of code can be remove as part of code clean up